oncodash-app/src/Oncoview.ts

```diff
@@ -219,13 +219,14 @@ function draw() {
   const blurMax = 3;
 
   const div = document.createElement("div");
-  const container = d3.select(div);
+  const container = d3.select(div)
+    .attr("id","oncowidget");
 
   // Canvas
   container.append('p').text('Oncoview');
   const svg = container.append("svg")
-    .style('width', svgWidth)
-    .style('height', svgHeight);
+    .attr("id","oncoview")
+    .attr('viewBox', '0 0 ' + svgWidth.toString() + " " + svgHeight.toString());
```

Comment:
`toString()` is redundant because string concatenation in JavaScript does automatic type conversion.

This can be made a bit more concise by using string interpolation (please note the backticks):
```js
`0 0 ${svgWidth} ${svgHeight}`
```

Or, if you want to emphasize that the viewbox is based on a numeric array:
```js
[0, 0, svgWidth, svgHeight].join(" ")
```